client/state/stats/lists/utils.js

```diff
@@ -657,6 +657,10 @@ export const normalizers = {
 				className: 'module-content-list-item-large',
 			};
 
+			if ( record.label.length === 0 ) {
+				record.label = translate( 'Untracked Author' );
```

Comment:
What do you think about combining this with line 652?
```
label: item.name || translate( 'Untracked Author' ),
```

Reply:
I'd second what @grzegorz-cp suggested - the data is from API, it would be more robust to just do an empty check like that. Otherwise for example, if the data isn't a string, L660 would error out.

Reply:
We could also use the [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing):

```javascript
label: item.name ?? translate( 'Untracked Author' ),
```

Reply:
> label: item.name ?? translate( 'Untracked Author' ),

Note: Empty strings would still be returned as empty string tho

Reply:
I find it interesting that an empty string evaluates to false in JavaScript. I would have expected the presence of a String object to be "truthy", which is why I hadn't considered using a [Logical OR](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR) here. Good to know!

Tested and confirmed that it does work for both empty and undefined values.